### PR TITLE
Allow service labels of jenkins to be configurable

### PR DESCRIPTION
**Pull request by waseem-h**
**What this PR does / why we need it**:
This PR allows us to add custom labels on jenkins service. This is needed for some kubernetes controllers like `exposecontroller` which looks for a specific label on services and exposes ingress if that label has a specific value.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
None
**Special notes for your reviewer**:
It would be great if this PR could be merged quickly. Thanks

### CODE REVIEW

File: stable/jenkins/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.17.0
+version: 0.17.1
```
Comment (davidkarlsen) [inline stable/jenkins/Chart.yaml]: new feature - use version 0.18.0

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by kahootali** [comment]
@davidkarlsen made the changes, kindly review it

**Comment by waseem-h** [comment]
Can you please review? @davidkarlsen I've made the changes requested

**Pull request closed without merging**